### PR TITLE
Write tests for yii\behaviors\TimestampBehavior

### PR DESCRIPTION
**Pull request by vladis84**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes

### CODE REVIEW

File: tests/framework/behaviors/TimestampBehaviorTest.php [hunk 1]
```diff
@@ -188,6 +188,41 @@ public function testUpdateRecordExpression()
...
+    {
+        ActiveRecordTimestamp::$behaviors = [
+            'timestamp' => [
+                'class' => TimestampBehavior::className(),
+                'value' => new Expression("strftime('%Y')"),
+            ],
+        ];        
+        $model = new ActiveRecordTimestamp();
+        
+        $this->setExpectedException('yii\base\InvalidCallException');
```
Comment (cebe) [inline tests/framework/behaviors/TimestampBehaviorTest.php]: can you explain why this expects an InvalidCallException?

Comment (Faryshta) [inline tests/framework/behaviors/TimestampBehaviorTest.php]: https://github.com/yiisoft/yii2/blob/master/framework/behaviors/TimestampBehavior.php#L129

the next line tries to update an attribute from a new record using the `touch()` method

Comment (cebe) [inline tests/framework/behaviors/TimestampBehaviorTest.php]: thanks!

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**